templates/lune/partials/product/product.snip.html

```diff
@@ -0,0 +1,175 @@
+{{#product}}
+  <main role="main" class="lune-page lune-page-product {{classes}}">
+    <div class="lune-page-product-wrapper md-col-9 mx-auto">
+
+      {{#hero}}
+        <section class="lune-product-section lune-product-section-hero relative z1 flex">
+          <div class="lune-hero-content col-12 md-mt0 flex flex-column items-center justify-center">
+            <div class="lune-hero-title-wrapper">
```

Comment:
The wrapper is needed as the positioning of the title is very different between mobile and desktop. I have however refactored it to make it more semantic